services/scan/src/precinct_scanner_app.ts

```diff
@@ -248,15 +250,15 @@ export async function buildPrecinctScannerApp(
       });
       return;
     }
-    store.setCurrentPrecinctId(bodyParseResult.ok().precinctId);
+    store.setPrecinctSelection(bodyParseResult.ok().precinctSelection);
     await configureInterpreter(interpreter, workspace);
     response.json({ status: 'ok' });
   });
 
-  app.delete<NoParams, Scan.DeleteCurrentPrecinctConfigResponse>(
+  app.delete<NoParams, Scan.DeletePrecinctSelectionConfigResponse>(
```

Comment:
Is this endpoint used anymore? I noticed you removed it from the tests

Reply:
It is being used, yes, I'll have to check where I removed it from the tests.

Reply:
I guess a more relevant question would be: is there a use case for clearing the VxScan precinct selection back to an unconfigured state? I can't really see why a user would want to do that personally.

Reply:
When you unconfigure the machine, you want it to return to a no precinct state. The precinct selection makes no sense outside of the context of the election definition, so if the election definition is removed, the precinct selection should be too.

Are you suggesting that we delete this endpoint and then just remove the precinct selection on zeroing out the scanner? That makes more sense to me, now that I'm thinking about it.

Reply:
Yeah, and I think that's actually what we already do. Just left another comment about it haha